packages/gatsby-dev-cli/src/watch.js

```diff
@@ -0,0 +1,51 @@
+const chokidar = require(`chokidar`)
+const _ = require(`lodash`)
+const fs = require(`fs-extra`)
+const syspath = require(`path`)
+
+const ignoreRegs = [/[\/\\]node_modules[\/\\]/i, /\.git/i, /[\/\\]src[\/\\]/i]
```

Comment:
Seems reasonable. @jquense could you make a quick PR for this to make sure it works for you locally? I'll just merge this PR now.